platforms/nuttx/src/px4/common/include/px4_platform/board_ctrl.h

```diff
@@ -46,11 +46,11 @@
 #define IOCTL_IDX_TO_BASE(x)	((((x) & 0xF) << 8) | _BOARDCTLIOCBASE)
 #define IOCTL_BASE_TO_IDX(x)	(((x) & 0x0F00) >> 8)
 
-#define _ORBIOCDEVBASE				IOCTL_IDX_TO_BASE(0)
+#define _ORBIOCDEVBASE			IOCTL_IDX_TO_BASE(0)
 #define _HRTIOCBASE				IOCTL_IDX_TO_BASE(1)
```

Comment:
while at it, maybe indent this as well :)

Reply:
Hmm, on my editor they are indented to the same TAB boundary, maybe there are mixed single whitespace + TABs here too